BezierKit/Library/PathComponent+Arc.swift

```diff
@@ -0,0 +1,166 @@
+//
+//  PathComponent+Arc.swift
+//  BezierKit
+//
+//  Created by ky1vstar on 15.10.2020.
+//  Copyright © 2020 Holmes Futrell. All rights reserved.
+//
+
+import Foundation
+
+extension PathComponent {
+    public convenience init(
+        ovalIn rect: CGRect,
+        numberOfCurves: Int = 4
+    ) {
+        let diameter = min(rect.width, rect.height)
```

Comment:
we should look at how this behaves with negative dimension rectangles. Core Graphics actually allows this and we should be sure to behave in the same way.